qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/QaTestApplication.kt

```diff
@@ -1,14 +1,15 @@
 package com.mapbox.navigation.qa_test_app
 
 import android.app.Application
+import com.mapbox.androidauto.MapboxCarApp
+import com.mapbox.navigation.qa_test_app.car.search.MapboxCarSearchApp
 
 class QaTestApplication : Application() {
 
     override fun onCreate() {
         super.onCreate()
 
-        // Uncomment when testing android-auto
-        // MapboxCarApp.setup(this)
-        // MapboxCarSearchApp.setup(this)
+         MapboxCarApp.setup(this)
```

Comment:
Should we comment this out before merging?
@kmadsen

Reply:
This is the only way to setup android auto at the moment, so that would be an issue with voice instructions.

But it doesn't play dual instructions anymore, thanks to https://github.com/mapbox/mapbox-navigation-android/pull/5846 🎉